ui/ecsconfig.py

```diff
@@ -480,25 +480,33 @@ def add_all():
         return None
 
     if l:
```

Comment:
Thanks for the clarification. I opened the issue #232 to deal with it in the future :)